src/test/emulators/extensions/validation.spec.ts

```diff
@@ -27,8 +48,29 @@ function getTestInstanceSpecWithAPI(instanceId: string, apiName: string): Instan
   };
 }
 
-describe("ExtensionsEmulator validation utils", () => {
-  describe(`${utils.getUnemulatedAPIs.name}`, () => {
+function testEmulatableBackend(predefinedTriggers: ParsedTriggerDefinition[]): EmulatableBackend {
```

Comment:
nit: fakeEmulatableBackend? sounds like a test

Reply:
Switched to getTestEmulatableBackend, since its not really a fake.